address.go

```diff
@@ -86,7 +86,7 @@ type ListAddressResult struct {
 	// additional responses can be fetched by updating the ListAddressOptions
 	// parameter's AfterID field with the ID of the last item in this object's
 	// Addresses field.
-	HasMore bool `json:"has_more,omitempty"`
+	PaginatedCollection
```

Comment:
Maintain it in one place rather than twenty? Part of the new `PaginatedCollection` base struct.

![image](https://user-images.githubusercontent.com/17054780/228334497-ec8add05-57de-4aef-adb1-7d391742e266.png)